blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/RollupBatchWriteRunnable.java

```diff
@@ -51,11 +53,12 @@ public void run() {
         } catch (Exception e) {
             LOG.warn("not able to insert rollups", e);
             executionContext.markUnsuccessful(e);
-
+        } finally {
+            executionContext.decrementWriteCounter(writeContexts.size());
+            rollupsPerBatch.update(writeContexts.size());
+            rollupsWriteRate.mark(writeContexts.size());
```

Comment:
I wanted to ask you this earlier. It feels redundant to have timer, meter and histogram around this piece of code. What do you think?